find_xml_file/find_xml_file.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+
+def find_xml_file(folder):
+    files = os.listdir(folder)
+
+    xml_file = None
+
+    for filename in files:
+        if filename.endswith(".xml"):
+            xml_file = filename
+            break
```

Comment:
Would we want to change this for a scenario where there are more than one xml files?

Reply:
Great suggestion, Sara!